packages/next/src/client/components/router-reducer/reducers/navigate-reducer.ts

```diff
@@ -199,7 +261,7 @@ export function navigateReducer(
   // The without optimistic tree case happens when there is no loading state, in that case we suspend in this reducer
 
   // forceOptimisticNavigation is used for links that have `prefetch={false}`.
-  if (forceOptimisticNavigation) {
+  if (false && forceOptimisticNavigation) {
```

Comment:
q for Tim/Seb: I've temporarily disabled to see if it made some tests fail. Tim mentioned that this maybe was not needed anymore since it's difficult to make that work well with parallel routes anyway.

currently the logic only really works for pages without them, is it worth keeping it around?

Reply:
If we have a prefetch, that didn't render any content, we still have the router tree there, right? So navigating to a dynamic page would still work as optimistic if it has a shared sibling that has a loading.js?

It's important the you can instant see the next page's loading state in this scenario but since we have the real router tree in this case, that should be fine.

I think maybe the remaining use case is that you navigating between siblings that themselves have loading.js states where there's `prefetch={false}` and we don't have the router tree yet. Maybe that's fine since it's somewhat of a deopt case anyway?